psydac/polar/c1_linops.py

```diff
@@ -89,7 +89,7 @@ def dot( self, v, out=None ):
         return out
 
     # ...
-    def toarray( self ):
+    def toarray( self, *, order='C' ):
```

Comment:
The `order` argument is not used by this function... Shouldn't you pass it to `np.zeros` on line 101?

Reply:
the same comment as above, it's not implemented , I added them because of codacy , we can either implement them or raise a not implemented error

Reply:
fixed